integration-tests/api/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "@opentelemetry/api": "^1.0.0",
-    "@opentelemetry/core": "^1.0.0",
+    "@opentelemetry/core": "1.26.0",
```

Comment:
This is an "interesting" change. Was this intentional?

Ah, I see. The `@opentelemetry/core` dep here was added after the previous release: https://github.com/open-telemetry/opentelemetry-js/pull/4880/files#diff-e4887234120ee9d59d36d48dbc50fcfc91d62eb3ad969aa43838da1f28b706a4

So, yes, I think this update is fine.